### PR TITLE
Add .zenodo.json

### PR DESCRIPTION
**Pull request by rouault**
Add a .zenodo.json to prepare for automatic DOI generation when releasing PROJ.

I've given a try with a tag created on that branch in my fork with the sandbox.zenodo.org service: https://sandbox.zenodo.org/record/994082 / https://sandbox.zenodo.org/account/settings/github/repository/rouault/PROJ

See GDAL discussion thread: https://lists.osgeo.org/pipermail/gdal-dev/2022-January/thread.html#55228

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            "affiliation": "Spatialys",
+            "name": "Rouault, Even",
+            "orcid": "0000-0002-5068-0476"
+        },
+        {
+            "affiliation": "Planet Labs Inc",
+            "name": "Warmerdam, Frank"
+        },
+        {
+            "affiliation": "SDFE.dk",
```
Comment (kbevers) [inline .zenodo.json]:
```suggestion
            "affiliation": "Danish Agency for Data Supply and Efficiency",
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            "name": "Evers, Kristian"
+        },
+        {
+            "affiliation": "SDFE.dk",
```
Comment (kbevers) [inline .zenodo.json]:
```suggestion
            "affiliation": "Danish Agency for Data Supply and Efficiency",
```

**Review by kbevers — CHANGES_REQUESTED** [review]

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            "affiliation": "SRI International",
+            "name": "Karney, Charles",
+            "orcid": "0000-0002-5006-5836"
+        },
+        {
+            "name": "Couwenberg, Sebastiaan"
+        },
+        {
+            "affiliation": "North Road",
+            "name": "Dawson, Nyall"
```
Comment (nyalldawson) [inline .zenodo.json]:
```suggestion
            "name": "Dawson, Nyall",
            "orcid": "0000-0001-9812-7584"
```

**Comment by rouault** [comment]
oops I had forgotten Jerry Evenden because github doesn't know him. I put USGS as his affiliation. Not sure how to handle someone who is deceased.

**Comment by mwtoews** [comment]
Updated orcid values for @QuLogic and @snowman2 .. hope these look correct!

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            "name": "Rouault, Even",
+            "orcid": "0000-0002-5068-0476"
+        },
+        {
+            "affiliation": "Planet Labs Inc",
+            "name": "Warmerdam, Frank"
+        },
+        {
+            "affiliation": "Danish Agency for Data Supply and Efficiency",
+            "name": "Evers, Kristian"
```
Comment (kbevers) [inline .zenodo.json]:
```suggestion
            "affiliation": "Danish Agency for Data Supply and Efficiency",
            "name": "Evers, Kristian",
            "orcid": "0000-0002-1310-4576"
```

**Comment by schwehr** [comment]
Thanks @rouault .  Curious to see how the community actually uses this.

**Comment by kbevers** [comment]
Do we have an actual DOI now? If yes, we've been asked to register it at https://wiki.osgeo.org/wiki/DOI

**Comment by rouault** [comment]
> Do we have an actual DOI now?

not yet. The github integration I've enabled requires a new release to be issued so that the all-version DOI is generated. I've been told that we could do a manual upload of a past release to do that (MapServer and QGIS have just done that), but I fail to see how the DOI record of the github integration would then be connected with the one obtained through manual upload. I fear that we'd have to do a manual upload each time we issue a new release, which would a no-go as far as I'm concerned. Maybe that's just my ignorance of the system.
I've just been trying to simulate that situation on sandbox.zenodo.org having created initially a fake upload in https://sandbox.zenodo.org/record/997579 and done a github release of https://github.com/rouault/test_zenodo/releases/tag/v1.0 in https://sandbox.zenodo.org/account/settings/github/repository/rouault/test_zenodo, but the later seems to be stalled.

**Comment by kbevers** [comment]
Alright, it will just have to wait then :-) Thanks for setting all of this up!

**Pull request merged**